src/dev/run_eslint.js

```diff
@@ -7,39 +7,54 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import yargs from 'yargs';
+import { run } from '@kbn/dev-cli-runner';
 
 import { eslintBinPath } from './eslint';
 
-let quiet = true;
-if (process.argv.includes('--no-quiet')) {
-  quiet = false;
-} else {
-  process.argv.push('--quiet');
-}
-
-const options = yargs(process.argv).argv;
 process.env.KIBANA_RESOLVER_HARD_CACHE = 'true';
 
-if (!options._.length && !options.printConfig) {
-  process.argv.push('.');
-}
-
-if (!process.argv.includes('--no-cache')) {
-  process.argv.push('--cache');
-}
-
-if (!process.argv.includes('--ext')) {
-  process.argv.push('--ext', '.js,.mjs,.ts,.tsx');
-}
-
-// common-js is required so that logic before this executes before loading eslint
-require(eslintBinPath); // eslint-disable-line import/no-dynamic-require
-
-if (quiet) {
-  process.on('exit', (code) => {
-    if (!code) {
-      console.log('✅ no eslint errors found');
+if (process.argv.includes('--help') || process.argv.includes('-h')) {
+  console.log(
+    "This is a wrapper around ESLint's CLI that sets some defaults - see Eslint's help for flags:"
+  );
+  require(eslintBinPath); // eslint-disable-line import/no-dynamic-require
+} else {
+  run(
+    ({ flags }) => {
+      if (process.argv.includes('--verbose')) {
+        process.argv.splice(process.argv.indexOf('--verbose'), 1);
+      }
+      flags._ = flags._ || [];
+      const quiet = !!flags._.quiet;
```

Comment:
Previously this was defaulting to quiet, was the change intentional?

Reply:
I was a bit confused by why the script was set up to log exactly when `--quiet` is set to true, but now I realize that quiet was mostly for ESLint to avoid logging warnings. I'll reinstate this somehow, thanks for warning!